### PR TITLE
doc: update release procedure

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: doc/source/developer/release.rst [hunk 1]
```diff
@@ -107,6 +106,16 @@ You can check with the following command whether you can login::
...
 If you can't login to packages.groonga.org, you must be registered ssh public key.
 
+Confirm the results of each test
+------------------------------
+
+We confirm the results of all the below tests before setting the tag to Mroonga.
+Because if we will find problems in Mroonga after setting the tag to it, we must release it again.
+
+* `GitHub Actions <https://github.com/mroonga/mroonga/actions>`_
+* `TravisCI <https://travis-ci.org/github/mroonga/mroonga>`_
```
Comment (kou) [inline doc/source/developer/release.rst:116]:
```suggestion
* `Travis CI <https://travis-ci.org/github/mroonga/mroonga>`_
```

File: doc/source/developer/release.rst [hunk 2]
```diff
@@ -163,27 +172,29 @@ Create Linux and Windows packages.
...
 
 Execute the following command::
 
-    % make download
-    % make build
-    % make upload
+    % rake apt:release
 
-After uploading the packages, update sign/metadata. See Groonga's release procedure about details.
+Now we finish build and upload packages to http://packages.groonga.org/.
```
Comment (kou) [inline doc/source/developer/release.rst:183]:
```suggestion
Now we finish build and upload packages to https://packages.groonga.org/.
```

File: doc/source/developer/release.rst [hunk 3]
```diff
@@ -163,27 +172,29 @@ Create Linux and Windows packages.
...
+However, these packages are unsigned. We sign packages by executing the below commands::
+
+    cd $PACKAGES_GROONGA_ORG_REPOSITORY
```
Comment (kou) [inline doc/source/developer/release.rst:186]:
```suggestion
    % cd $PACKAGES_GROONGA_ORG_REPOSITORY
```

File: doc/source/developer/release.rst [hunk 4]
```diff
@@ -193,30 +204,57 @@ You can install packages via Groonga PPA on launchpad.net::
...
+    % cd packages
 
 Execute the following command::
 
-    % make download
-    % make build
-    % make upload
+    % rake yum:release
+
+Now we finish build and upload packages to http://packages.groonga.org/.
```
Comment (kou) [inline doc/source/developer/release.rst:215]:
```suggestion
Now we finish build and upload packages to https://packages.groonga.org/.
```

File: doc/source/developer/release.rst [hunk 5]
```diff
@@ -193,30 +204,57 @@ You can install packages via Groonga PPA on launchpad.net::
...
+However, these packages are unsigned. We sign packages by executing the below commands::
 
-After uploading the packages, update sign/metadata. See Groonga's release procedure about details.
+    cd $PACKAGES_GROONGA_ORG_REPOSITORY
```
Comment (kou) [inline doc/source/developer/release.rst:218]:
```suggestion
    % cd $PACKAGES_GROONGA_ORG_REPOSITORY
```

File: doc/source/developer/release.rst [hunk 6]
```diff
@@ -193,30 +204,57 @@ You can install packages via Groonga PPA on launchpad.net::
...
 
 For windows packages, we use `AppVeyor CI <https://ci.appveyor.com/project/groonga/mroonga>`_ artifacts files.
+We upload them to `GitHub release page <https://github.com/mroonga/mroonga/releases>`_ .
 
 Upload documents
 ----------------
 
 1. Clone mroonga.github.com repository
-2. Execute ``make update-document``
+2. Execute ``make update-document`` in $MROONGA_CLONE_DIR
```
Comment (kou) [inline doc/source/developer/release.rst:231]:
```suggestion
2. Execute ``make update-document`` in ``$MROONGA_CLONE_DIR``
```

File: doc/source/developer/release.rst [hunk 7]
```diff
@@ -193,30 +204,57 @@ You can install packages via Groonga PPA on launchpad.net::
...
 3. Commit changes in mroonga.github.com repository && push them
 
+Update blog(Mroonga blog)
+-------------------------
+
+We update the below files.
+
+* $MROONGA_GITHUB_COM_PATH/ja/_posts/(the date of release)-mroonga-(version).md
```
Comment (kou) [inline doc/source/developer/release.rst:239]:
```suggestion
* ``$MROONGA_GITHUB_COM_PATH/ja/_posts/(the date of release)-mroonga-(version).md``
```

File: doc/source/developer/release.rst [hunk 8]
```diff
@@ -193,30 +204,57 @@ You can install packages via Groonga PPA on launchpad.net::
...
+* $MROONGA_GITHUB_COM_PATH/en/_posts/(the date of release)-mroonga-(version).md
```
Comment (kou) [inline doc/source/developer/release.rst:240]:
```suggestion
* ``$MROONGA_GITHUB_COM_PATH/en/_posts/(the date of release)-mroonga-(version).md``
```

**Comment by komainu8** [comment]
@kou Thank you for your comments.
I've fixed them.

**Pull request merged**